examples/mandelbrot/src/lib.rs

```diff
@@ -126,9 +126,14 @@ mod tests {
         }
 
         assert_eq!(v_expected.len(), 3 * w * h);
-        verify(w, h, &v_simd, &v_expected);
-        verify(w, h, &v_par_simd, &v_expected);
-        #[cfg(feature = "ispc")]
-        verify(w, h, &v_ispc, &v_expected);
+        #[cfg(not(target_feature = "fma"))]
```

Comment:
Why isn't `is_x86_feature_detected!` used here ?

Reply:
Some people might have FMA support on their CPU at runtime, but would like to run this test. They can compile without `target_feature = "+fma"`.

Using runtime detection would mean that even if the code was built without fma (so the test should pass), the code wouldn't be tested if the host had FMA.